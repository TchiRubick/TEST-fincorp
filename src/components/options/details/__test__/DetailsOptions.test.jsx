import React from 'react';
import ReactDom from 'react-dom';
import { cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

import DetailsOptions from '../DetailsOptions';

afterEach(cleanup);

it('renders good', () => {
  const div = document.createElement('div');

  ReactDom.render(<DetailsOptions title="test">test</DetailsOptions>, div);
});

it('as expected snapshot 1', () => {
  const tree = renderer.create(<DetailsOptions title="test">test</DetailsOptions>).toJSON();
  expect(tree).toMatchSnapshot();
});
