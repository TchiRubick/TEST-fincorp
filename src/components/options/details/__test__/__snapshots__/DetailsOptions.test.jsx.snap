// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`as expected snapshot 1 1`] = `
<div
  className="comp-details-options"
>
  <div
    className="comp-details-options__icon"
  >
    <img
      alt="tick"
      className="comp-details-options__icon-img"
      src="tick.png"
    />
  </div>
  <div
    className="comp-details-options__text"
  >
    <span
      className="comp-details-options__text-title"
    >
      test
    </span>
    <span
      className="comp-details-options__text-sub"
    >
      test
    </span>
  </div>
</div>
`;
